Resources/Prototypes/Entities/Structures/Storage/Canisters/gas_canisters.yml

```
@@ -72,7 +72,7 @@
         fix1:
           shape:
             !type:PhysShapeAabb
-            bounds: "-0.25,-0.25,0.25,0.25"
+            bounds: "-0.3,-0.3,0.3,0.3" # mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)